dotnet/src/SemanticKernel.Abstractions/Data/RecordAttributes/VectorStoreRecordVectorAttribute.cs

```diff
@@ -31,6 +31,17 @@ public VectorStoreRecordVectorAttribute(int Dimensions)
         this.Dimensions = Dimensions;
     }
 
+    /// <summary>
+    /// Initializes a new instance of the <see cref="VectorStoreRecordVectorAttribute"/> class.
+    /// </summary>
+    /// <param name="Dimensions">The number of dimensions that the vector has.</param>
+    /// <param name="DistanceFunction">The distance function to use when comparing vectors.</param>
+    public VectorStoreRecordVectorAttribute(int Dimensions, string? DistanceFunction)
```

Comment:
I think if we add this, we should re-order the next constructor's parameters to match this ordering.

Reply:
Implemented it in separate PR: https://github.com/microsoft/semantic-kernel/pull/9260